diff --git a/node_modules/hexo-theme-next/layout/_macro/post-collapse.njk b/node_modules/hexo-theme-next/layout/_macro/post-collapse.njk
index 4c39119..ce7605b 100644
--- a/node_modules/hexo-theme-next/layout/_macro/post-collapse.njk
+++ b/node_modules/hexo-theme-next/layout/_macro/post-collapse.njk
@@ -33,6 +33,26 @@
         {%- endif %}
       </div>
 
+      {%- if post.categories and post.categories.length and theme.post_meta.categories %}
+    <span class="post-meta-item">
+      <span class="post-meta-item-icon">
+        <i class="far fa-folder"></i>
+      </span>
+      <span class="post-meta-item-text">{{ __('post.in') }}</span>
+      {%- for cat in post.categories.toArray() %}
+        <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
+          <a href="{{ url_for(cat.path) }}" itemprop="url" rel="index"><span itemprop="name">{{ cat.name }}</span></a>
+        </span>
+
+        {%- set cat_length = post.categories.length %}
+        {%- if cat_length > 1 and loop.index !== cat_length %}
+          {{ __('symbol.comma') }}
+        {%- endif %}
+      {%- endfor %}
+    </span>
+  {%- endif %}
+
+
       {{ post_gallery(post.photos) }}
     </header>
   </article>
